### PR TITLE
Update egg-team-fortress-2-classic.json

### PR DESCRIPTION
**Pull request by TheKing1543**
Made it work, lol.

# Description

The egg didn't work so I sorted it out with the help of a Team Fortress 2 Classic developer.

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [ ] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:
 Because I just forked the repo to fix the TF2C bug.

* [X] You verify that the start command applied does not use a shell script
  * [ ] If some script is needed then it is part of a current yolk or a PR to add one
* [X] The egg was exported from the panel

### CODE REVIEW

**Comment by parkervcp** [comment]
@TheKing1543 What was the exact change made here?

**Comment by TheKing1543** [comment]
The source multiplayer needs a folder named tf2classic to work, that the egg was simply not making, also, the zip format that the tf2c devs use is not unzipable (at least from my server), and it lead to errors unzipping, so they worked up a solution and gave me another link to a zip that is actually usable

**Comment by TheKing1543** [comment]
I'll explain further

- When the server installs, there needs to be a "tf2classic" folder inside root to make SourceSDK load TF2 Classic, one of the problem was, that it was expanding the TF2C folder inside root, merging with SteamCMD and messing everything up.

- There was another error, at least with my server, "need pk compat". That was when extracting the zip file. So the developer gave me another link to a zip file that did not require pk compat.

**Comment by QuintenQVD0** [comment]
You are not allowed to change the authoe mail! Pleasd change it back.

File: game_eggs/steamcmd_servers/team_fortress_2_classic/egg-team-fortress-2-classic.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2023-06-02T19:04:54+02:00",
+    "exported_at": "2023-07-13T23:07:08+01:00",
     "name": "Team Fortress 2 Classic",
-    "author": "eggs@scattergun.io",
```
Comment (parkervcp) [inline game_eggs/steamcmd_servers/team_fortress_2_classic/egg-team-fortress-2-classic.json:9]: This needs to stay the same

**Review by parkervcp — CHANGES_REQUESTED** [review]

**Review by parkervcp — APPROVED** [review]

**Pull request merged**